README.md

```diff
@@ -156,7 +159,7 @@ cd mava
 pip install -e .
 ```
 
-We have tested `Mava` on Python 3.9. Note that because the installation of JAX differs depending on your hardware accelerator,
+We have tested `Mava` on Python 3.11 and 3.12, but earlier versions may also work. Note that because the installation of JAX differs depending on your hardware accelerator,
```

Comment:
Bit of a nit but could we please add a `python 3.11` linter Github workflow as well please?